### ISSUE TITLE
[map] Add missing counts of misses and evictions to LocalMapStats

### ISSUE THREAD

**Issue by kennymacleod** [hazelcast/hazelcast#3009]
The `LocalMapStats` interface has a `getHits` method, but this isn't much use in and of itself. It needs a corresponding `getMisses` method to give it come context. This method used to be present in hazelcast 2.x, but seems to have been inadvertently deleted in 3.x.

While we're at it, we also need a `getEvictions` statistic to get a count of the number of elements that have been evicted (due to TTL or too-many-items, for example).

**Comment by blakeday** [comment]
Also, how do you get operations per second in the latest version?

**Comment by PhillipMek** [comment]
@mmedenjak, is any updates on this issue? It will be cool to add eviction and expiration events monitoring in Management Center.

**Comment by mmedenjak** [comment]
No, there was not much demand for this and we worked on higher priority items but we're happy to accept a PR from the community.

**Comment by JaegerJochimsen** [comment]
@mmedenjak Hi there, are there any updates?
